### PR TITLE
Node's network based header background

### PR DESCRIPTION
**Pull request by rottingcorpse**
Changes MEW header background based on selected node's network.

Default: blue'ish gradient for ETH
ETC: Green gradient (green and blue color values swapped for header gradient colors)

### CODE REVIEW

File: common/components/Header/index.jsx [hunk 1]
```diff
@@ -22,7 +22,7 @@ export default class Header extends Component {
 
     return (
       <div>
-        <section className="bg-gradient header-branding">
+        <section className="bg-gradient header-branding {selectedNode.network}">
```
Comment (NessDan) [inline common/components/Header/index.jsx]: The class name feels too generic, does attaching a `branding-node-{selectedNode.network}` sound good?

File: common/assets/styles/etherwallet-utilities.less [hunk 2]
```diff
@@ -23,8 +23,12 @@
 .bg-gradient {
   background: @ether-navy;
   background: linear-gradient(149deg, #132a45, #143a56, #21a4ce, #19b4ad);
+  &.ETC {
```
Comment (NessDan) [inline common/assets/styles/etherwallet-utilities.less]: ETC doesn't exist in the config data file, is there plans to add ETC to MEW?

File: common/components/Header/index.jsx (hunk 1 above)
Comment (crptm) [inline common/components/Header/index.jsx]: This wouldnt interpolate

File: common/assets/styles/etherwallet-utilities.less (hunk 2 above)
Comment (crptm) [inline common/assets/styles/etherwallet-utilities.less]: yep

**Comment by dternyak** [comment]
@crptm / @NessDan This PR is to claim a bounty against V3 and as such should be submitted at https://github.com/kvhnuke/etherwallet

**Pull request closed without merging**